connstr/README.md

```diff
@@ -0,0 +1,118 @@
+# Connect
+
+Connect to Oracle Database using `sql.Open("godror", dataSourceName)` where
+`dataSourceName` contains options such as the user credentials, the database
+connection string, and other configuration settings.   It should be a
+[logfmt](https://brandur.org/logfmt)-encoded  parameter list.   For example:
+
```

Comment:
Now I've testing it, what about adding a 'real' example to make it easier for newbies to cut & paste? Perhaps change the example to be something like this?:

----------
For example:
```
db, err := sql.Open("godror", `user="scott" password="tiger" connectString="dbhost:1521/orclpdb1"`)
```
Other connection and driver options can also be used:

```
db, err := sql.Open("godror", `user="scott" password="tiger" connectString="dbhost:1521/orclpdb1?connect_timeout=2" poolSessionTimeout=42s configDir="/opt/oracle/configdir" heterogeneousPool=false standaloneConnection=false`)
```

---------

@tgulacsi since I don't use your older syntaxes, I find not allowing line breaks in this syntax is a bit painful.